mapper/transaction.go

```diff
@@ -62,7 +73,75 @@ func Transaction(
 
 	traceOps := traceOps(flattenedTrace, len(feeOps))
 	ops = append(ops, traceOps...)
+	for _, transferLog := range transferLogs {
```

Comment:
We shouldn't even be fetching logs in standard mode (as it will add latency).

Reply:
nvm saw this is handled below...may make sense to put a comment around this.

Reply:
Yep, we shouldn't have any logs.  Originally had it more explicit with an if statement, but the linter said my code was too complex :laughing: and I had to remove it

Reply:
Also, in standard mode we will have logs if there's a non-empty token whitelist